src/Network/Receive/kRO/Sakexe_2004_07_13a.pm

```diff
@@ -20,29 +20,9 @@ package Network::Receive::kRO::Sakexe_2004_07_13a;
 use strict;
 use base qw(Network::Receive::kRO::Sakexe_2004_07_05a);
 
-use Log qw(message warning error debug);
-use Utils qw(getTickCount getCoordString);
-
-# TODO: maybe we should try to not use globals in here at all but instead pass them on?
-use Globals qw($char);
-
-
 sub new {
 	my ($class) = @_;
 	return $class->SUPER::new(@_);
 }
 
-=pod
-//2004-07-13aSakexe
```

Comment:
Dont del it . !!!

Reply:
we already have it in packets in table folders
a lot of this comment is only duplicated

Reply:
:thinking: